_posts/2019-03-28-pulp-3-rc-release.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Pulp 3 Core RC Release
+author: David Davis
+---
+
+Today we are excited to announce an important milestone of the Pulp 3 project, the release of
```

Comment:
This new content looks great. I just reread the whole post and it looks good to me. Ty for writing this!